datadog_checks_dev/datadog_checks/dev/tooling/commands/meta/snmp/build_traps_db.py

```diff
@@ -0,0 +1,294 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import json
+import os
+from contextlib import ExitStack
+from functools import lru_cache
+
+import click
+import yaml
+from pysmi import error
+from pysmi.codegen import JsonCodeGen
+from pysmi.compiler import MibCompiler
+from pysmi.parser import SmiV1CompatParser
+from pysmi.reader import getReadersFromUrls
+from pysmi.searcher import AnyFileSearcher
+from pysmi.writer import FileWriter
+
+from datadog_checks.dev import TempDir
+
+from ...console import CONTEXT_SETTINGS, abort, echo_failure, echo_info, echo_success, echo_warning
+from .constants import MIB_SOURCE_URL
+
+# Unique identifiers of traps in json-compiled MIB files.
+NOTIFICATION_TYPE = 'notificationtype'
+
+
+@click.command(
+    context_settings=CONTEXT_SETTINGS,
+    short_help='Generate a traps database that can be used by the '
+    'Datadog Agent for resolving Traps OIDs to readable strings.',
+)
+@click.option(
+    '--mib-sources',
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities. -->
**💭 thought**
Same thought as per the command name, should we have coherent names across profile and traps db builder/ generator ?

Reply:
`mib-sources` is more explicit than `source`, I like it

Reply:
That's a fair point, but `--mib-sources` is more explicit and has some benefits over "source" so I'd rather keep it like this. Might be good to harmonize the various commands arguments though 💭